twint/user.py

```diff
@@ -46,7 +48,9 @@ def User(ur):
     _usr.is_private = ur['data']['user']['legacy']['protected']
     _usr.is_verified = ur['data']['user']['legacy']['verified']
     _usr.avatar = ur['data']['user']['legacy']['profile_image_url_https']
-    _usr.background_image = ur['data']['user']['legacy']['profile_banner_url']
+    _usr.background_image = ""
+    if 'profile_banner_url' in ur['data']['user']['legacy']:
+        _usr.background_image = ur['data']['user']['legacy']['profile_banner_url']
```

Comment:
there is already a PR (#1005 ) for `user.py ` which takes care of these exact issues of `profile_banner_url`, `url` and  `malformed json` check at line 21.

Reply:
I see. Sorry about that! How would you like me to handle this? Remove these changes now or wait until PR #1005 is merged in?

Reply:
Just make your changes consistent with mine (user.py file, rest of the portion is fine), so my changes don't get overwritten when your PR gets merged into master.